components/molecules/Table.jsx

```diff
@@ -124,7 +124,7 @@ class Table extends Component {
                                         onClick={handleHeadingClick}
                                         key={reactKey}
                                         isSecondary={isSecondary}
-                                        className={defaultCursor && '-cursor--default'}
+                                        className={defaultCursor ? '-cursor--default' : undefined}
```

Comment:
Nit: It's probably just semantics, but I prefer using an empty string so that the props are always using the same type. Feel free to leave it this way, tho.

Reply:
ahh ya, so when you do `undefined` React ignores the prop completely, which is nice since it's also not displayed in the Storybook source.